internal/conf/conf.go

```diff
@@ -285,7 +286,13 @@ func (conf *Conf) CheckAndFillMissing() error {
 		conf.ReadBufferCount = 512
 	}
 	if (conf.ReadBufferCount & (conf.ReadBufferCount - 1)) != 0 {
-		return fmt.Errorf("'ReadBufferCount' must be a power of two")
+		return fmt.Errorf("'readBufferCount' must be a power of two")
+	}
+	if conf.UDPMaxPayloadSize == 0 {
+		conf.UDPMaxPayloadSize = 1500
+	}
+	if conf.UDPMaxPayloadSize > 1500 {
```

Comment:
Is that OK? Why are you blocking this? I mean, it should work in environments with support "Don't fragment" = 0. As seen in AWS: ETH > 1500 but fragmented IP.